AlarmItem[] alarms
---
bool success
string message